src/include/duckdb/main/config.hpp

```diff
@@ -276,6 +310,8 @@ struct DBConfig {
 	DUCKDB_API IndexTypeSet &GetIndexTypes();
 	static idx_t GetSystemMaxThreads(FileSystem &fs);
 	void SetDefaultMaxMemory();
+	void SetDefaultMaxSwapSpace(optional_ptr<DatabaseInstance> db);
```

Comment:
This can be removed as well once we store this in the temp directory handle instead